src/libsyntax/parse/lexer/mod.rs

```diff
@@ -1408,13 +1133,9 @@ impl<'a> StringReader<'a> {
                     // lifetimes shouldn't end with a single quote
```

Comment:
Pre-existing inaccuracy in error recovery few lines above - `ident_start(Some(c2)) || c2.is_numeric()` probably wants to be `ident_continue(c2)`.
(With the non-fatal error on `!ident_start(c2)` instead of `c2.is_numeric()` below.)

Reply:
Yeah, `is_numeric` makes no sense, `¾` is numeric...

Reply:
For the reference, special casing of numbers was added in https://github.com/rust-lang/rust/pull/58876/files